### DISCUSSION TITLE
Student Verification Denied 10 Times

### DISCUSSION THREAD

**Discussion by DFMlittleD** [community/community#51831] [GitHub Education] [answered]
I have tried 10 times to submit my application, including submitting a photo of my student ID, a photo of my student ID with English translation, an online verification report from the Ministry of Education, and a preliminary manually translated online verification report from the Ministry of Education, during which I did my best to complete my personal information, put the pinyin of my name in my nickname and profile, filled in everything except my personal website (which I don't have) and social media, and finally submitted a photo of myself holding my student ID for verification, yet I still need to complete my personal information, and I really don't know what else to fill in.
![Screenshot 2023-04-03 183707](https://user-images.githubusercontent.com/129506197/229486159-08595944-01a2-42d5-9035-455c0a942b10.png)

**Comment by DFMlittleD** [comment, marked as answer]
Student ID card can not pass, hold up a small piece of paper with ID, school and school number can pass .... .OK, Github

**Reply by solo-driven** [reply]
So you bypassed it just by writing on sheet of paper your id and ... ? Did you get verified?

**Reply by DFMlittleD** [reply]
Yeah...A paper with my GitHub Name, my own name, school, student ID (what is school number? I must be too speechless yesterday...) and my face . Then I got vertified instantly.

**Reply by samus-aran** [reply]
Hi @solo-driven & @DFMlittleD 

Just want to clarify it sounds like there was some translation needed. 

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.

@DFMlittleD Was your evidence in a non English language? 🤔

**Reply by mutigaj1** [reply]
what if i dont have a school ID?

**Reply by jja8041** [reply]
I have to concur. I gave it everything short of my dna sequence and it acts like I'm some sort of crook. I'm happy for those who got through but my laptop camera is broken so the system hates me for trying to upload a picture from my phone. *Nixon Voice* "I am not a crook!"

**Reply by ThatRichDude** [reply]
I am on the same boat. Too many rejections. Make no sense!
